### ISSUE TITLE
[Improvement] File with incorrect header / missing LICENSE information

### ISSUE THREAD

**Issue by justinmclean** [datastrato/gravitino#2800]
### What would you like to be improved?

This file looks to be a 3rd party file from https://github.com/JohnReedLOL/kafka-streams

catalogs/catalog-messaging-kafka/src/test/java/com/datastrato/gravitino/catalog/kafka/embeddedKafka/ZooKeeperEmbedded.java

### How should we improve?

Restore the header and remove the Datatstrato header and add license and file information to LICENSE. As this looks to be under the Apache license and has a NOTICE file it will need to be mentioned there as well. https://github.com/JohnReedLOL/kafka-streams/blob/master/NOTICE

**Comment by justinmclean** [comment]
@mchades Can you please confirm if this is the case.

**Comment by mchades** [comment]
Mainly from [this](https://github.com/confluentinc/kafka-streams-examples/blob/master/src/test/java/io/confluent/examples/streams/zookeeper/ZooKeeperEmbedded.java), could you please help me confirm if there are any license issues with this project?

**Comment by justinmclean** [comment]
Yes, we need to know where it originally came from (note the Copyright Confluent Inc.) and if it had a NOTICE file. It needs to be added to the LICENSE files and NOTICE file if the original had a NOTICE file.

**Comment by justinmclean** [comment]
The original header needs to be replaced, as someone else is the copyright owner.
